test/unit/resources/kernel_module_test.rb

```diff
@@ -11,18 +11,48 @@
     _(resource.loaded?).must_equal true
   end
 
-  it 'verify kernel_module parsing' do
+  it 'verify kernel_module parsing loaded' do
     resource = load_resource('kernel_module', 'bridges')
     _(resource.loaded?).must_equal false
   end
 
   it 'verify kernel_module parsing' do
+    resource = load_resource('kernel_module', 'bridge')
+    _(resource.enabled?).must_equal true
+  end
+
+  it 'verify kernel_module parsing' do
     resource = load_resource('kernel_module', 'dhcp')
     _(resource.loaded?).must_equal false
   end
 
+  it 'verify kernel_module parsing disabled' do
+    resource = load_resource('kernel_module', 'bridge')
+    _(resource.disabled?).must_equal false
+  end
+
+  it 'verify kernel_module parsing disabled' do
+    resource = load_resource('kernel_module', 'bridges')
+    _(resource.disabled?).must_equal false
+  end
+
+  it 'verify kernel_module disabled via /bin/true' do
+    resource = load_resource('kernel_module', 'floppy')
+    _(resource.disabled_via_bin_true?).must_equal false
+  end
+
+  it 'verify kernel_module disabled via /bin/false' do
```

Comment:
This is a good test to make sure that the `disabled_via_bin_false?` method returns false for an unlisted module, but we still need a test for a listed module.

I'd rename this test to "verify an unlisted kernel_module is not disabled via /bin/true" and leave the rest of it alone, and then add another test using this existing title ("verify kernel_module disabled via /bin/true") and test module `sstfb` and checking that the method returns `true`